openlibrary/core/processors/readableurls.py

```diff
@@ -43,7 +43,8 @@ def __call__(self, handler):
         web.ctx.path = real_path
         web.ctx.fullpath = web.ctx.path + web.ctx.query
         out = handler()
-        V2_TYPES = ['works', 'books', 'people', 'publishers', 'languages']
+        V2_TYPES = ['works', 'books', 'people', \
+            'authors', 'publishers', 'languages']
```

Comment:
A `\` actually isn't required here because the interpreter is smart enough to detect the beginning and end of the list. Splitting on multiple lines is sufficient.

This isn't the case with strings, e.g. 
```
x = "this would require a " \
  "slash in order to concat"

y = ("however, by wrapping in parens, similar to the list, "
  "no slash is required")
```

Reply:
thanks for the tip! fixed!